### PR TITLE
Tech debt/add af sdk

### PR DESCRIPTION
**Pull request by kcw-grunt**
Adding analytics package to more accurately track installs

### CODE REVIEW

File: app/src/main/java/com/breadwallet/tools/util/Utils.java [hunk 1]
```diff
@@ -268,6 +268,19 @@ else if (name == PartnerNames.OPSALL) {
                     }
                     return opsString.replaceAll("\\s+","");
                 }
+                else if (name == PartnerNames.AFDEVID) {
```
Comment (kcw-grunt) [inline app/src/main/java/com/breadwallet/tools/util/Utils.java:271]: Allows business logic to retrieve the proper key

File: app/src/main/java/com/breadwallet/BreadApp.java [hunk 2]
```diff
@@ -47,27 +42,19 @@ public class BreadApp extends Application {
...
-        }
-
+        /// DEV:  Top placement req.
+        boolean enableCrashlytics = !Utils.isEmulatorOrDebug(this);
         FirebaseCrashlytics.getInstance().setCrashlyticsCollectionEnabled(enableCrashlytics);
         AnalyticsManager.init(this);
         AnalyticsManager.logCustomEvent(BRConstants._20191105_AL);
-        AppsFlyerLib.getInstance().init("XXXX", null, this);
 
-        WindowManager wm = (WindowManager) getSystemService(Context.WINDOW_SERVICE);
```
Comment (kcw-grunt) [inline app/src/main/java/com/breadwallet/BreadApp.java:65]: cut it down to 1 line 🥳

Comment (josikie) [reply app/src/main/java/com/breadwallet/BreadApp.java:65]: Wahh nice

**Review by josikie — APPROVED** [review]
Cool! Thanks @kcw-grunt

**Pull request merged**